src/components/button/_button.scss

```diff
@@ -112,7 +105,7 @@
     padding: em($govuk-spacing-scale-1, 24px) em($govuk-spacing-scale-6, 24px) em($govuk-spacing-scale-1, 24px) em($govuk-spacing-scale-4, 24px);
     background-image: file-url("icon-pointer.png");
 
-    @include device-pixel-ratio() {
+    @include govuk-h-device-pixel-ratio() {
```

Comment:
Linter is complaining about the empty-args. We can update this line (and the other one is `_breadcrumbs.scss`) or we can adjust the linter if we're not happy with him picking that up.